cypress/e2e/functional/researcher_permission_forms.spec.js

```diff
@@ -0,0 +1,113 @@
+    TEST_PERMISSION_FORM_URL  = 'concord.org'
+    // TODO: I don't know the password for c.RESEARCHER_USERNAME (i.e. 'researcher'), so I've created my own, for now
+    , RESEARCHER_USERNAME = 'atestresearcherforcypresstesting'
+;
+
+context("Researcher uses permission forms", () => {
+
+    before(function() {
+        cy.visit(c.LEARN_PORTAL_BASE_URL); // Visit LEARN Portal home page
+        // cy.login(c.RESEARCHER_USERNAME, c.RESEARCHER_PASSWORD); // Login as researcher user
+        cy.login(RESEARCHER_USERNAME, c.RESEARCHER_PASSWORD); // Login as researcher user
+    });
+    
+    after(function() {
+        cy.clearAllCookies();
+        cy.ge
+    });
+
+    it("Tests Permission forms", () => {
+        cy.log('Verify, and then click, the Admin link');
+        pf.verifyAdminLink();
+        pf.clickAdminLink();
+
+        cy.log('Verify, and then click, the Permission Forms link');
+        pf.verifyPermissionFormsLink();
+        pf.clickPermissionFormsLink();
+
+        cy.log('Verify that both links exist; but only the Create / Manage Project Permission Forms link is enabled');
+        pf.verifyManageStudentPermissionsLink();
+        pf.verifyManageStudentPermissionsLinkEnabled(false);
+        pf.verifyCreateManageProjectPermissionFormsLink();
+        pf.verifyCreateManageProjectPermissionFormsLinkEnabled();
+
+        cy.log('Click the Create / Manage Project Permission Forms link; then verify that only the Manage Student Permissions link is enabled');
+        pf.clickCreateManageProjectPermissionFormsLink();
+        pf.verifyManageStudentPermissionsLinkEnabled();
+        pf.verifyCreateManageProjectPermissionFormsLinkEnabled(false);
+
+        cy.log('Click the Create New Permission Form button, fill in values in the dialog; but then click Cancel');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME, 'google.com', true);
+
+        cy.log('Fill in the Create New Permission Form dialog again, but this time, click the Save button');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+
+        cy.log('Create more Permission Forms, to be used to test the Archive, Unarchive & Delete buttons');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'4', 'learn.portal.staging.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+            
+        cy.log("Select a Project from the dropdown (on the 'Create New Permission Form' tab)");
+        pf.selectCreateManagePermissionsProject(TEST_PROJECT_NAME);
```

Comment:
@johnTcrawford once we create the permission form, the project name that is selected will be displayed by default in the Create/ Manage Project Permission Forms tab. So, we don't want to select that project again.

Reply:
It does no harm to select it again, though you're right that it is unnecessary. Perhaps I'll replace this with a check that the Project name shown is the one we expect. (For now, I just added a TODO in the code for this.)